src/__smoke-test__/ingestion-integ.spec.ts

```diff
@@ -447,3 +448,45 @@ test('when event with custom attributes is sent then the event is ingested', asy
     );
     expect(isIngestionCompleted).toEqual(true);
 });
+
+test('when INP event is sent then event is ingested', async ({ page }) => {
```

Comment:
nit: out of scope of this PR, but we should probably clean the structure of the code to prevent having to copy paste the same smoke tests in two different places

Reply:
agreed